features/support/hooks.js

```diff
@@ -71,7 +68,7 @@ module.exports = function () {
 }
 
 function handlePrimusConnection (ctx, client) {
-  client.on('data', function (message) {
+  client.on('data', (message) => {
     var shortHash
```

Comment:
ah! this file. I got to some of them and managed to change them, but some of them gave me a hard time (but I think only the function declarations). I'll look into this.